examples/opt/armv7m/armv7m_simple0_opt_m7.s

```diff
@@ -1,69 +1,75 @@
+                                        // Instructions:    27
+                                        // Expected cycles: 27
+                                        // Expected IPC:    1.00
+                                        //
+                                        // Cycle bound:     27.0
+                                        // IPC bound:       1.00
+                                        //
+                                        // Wall time:     0.22s
+                                        // User time:     0.22s
+                                        //
+                                        // ----- cycle (expected) ------>
+                                        // 0                        25
+                                        // |------------------------|----
+        ldr r4, [r0, #4]                // *............................. // @slothy:reads=a
+        add r4, r2, r4                  // .*............................
+        eor.w r4, r4, r3                // ..*...........................
+        smlabt r2, r2, r2, r4           // ..*...........................
+        asrs r2, r2, #1                 // ....*.........................
+        str r2, [r0, #4]                // ....*......................... // @slothy:writes=a
+        ldm r0, {r11,r12,r14}           // .....*........................ // @slothy:reads=a
+        add r2, r12, r11                // ........*.....................
+        eor.w r2, r2, r14               // .........*....................
+        smlabt r2, r12, r12, r2         // .........*....................
+        asrs r2, r2, #1                 // ...........*..................
+        str r2, [r0, #4]                // ...........*.................. // @slothy:writes=a
+        ldm r0, {r11,r12,r14}           // ............*................. // @slothy:reads=a
+        add r2, r12, r11                // ...............*..............
+        eor.w r2, r2, r14               // ................*.............
+        smlabt r2, r12, r12, r2         // ................*.............
+        asrs r2, r2, #1                 // ..................*...........
+        str r2, [r0, #4]                // ..................*........... // @slothy:writes=a
+        ldm r0, {r11,r12,r14}           // ...................*.......... // @slothy:reads=a
+        add r2, r12, r11                // ......................*.......
+        eor.w r14, r2, r14              // .......................*......
+        smlabt r6, r12, r12, r14        // .......................*......
+        ldrd r4, r2, [r14, #8]          // ........................*.....
+        str r4, [r0]                    // ........................*.....
+        asrs r2, r6, #1                 // .........................*....
+        str r2, [r0, #4]                // .........................*.... // @slothy:writes=a
+        ldm r0, {r0,r1,r2,r3}           // ..........................*...
```

Comment:
Your output shows that you have not turned on fusion. So the code you have commited isn't actually used. 
You need to add a call to `fusion_region` into the corresponding example in `example.py`